examples/stm32/xively_demo_stm32f4xx_nucleo_wifi/Src/main.c

```diff
@@ -763,14 +763,17 @@ static inline void print_user_config_debug_banner( void )
     printf( "\r\n|********************************************************|" );
     printf( "\r\n|               User Configuration Routine               |" );
     printf( "\r\n|               --------------------------               |" );
-    printf( "\r\n| WARNING: Do NOT copy-paste long strings to your serial |" );
-    printf( "\r\n| -------  terminal. Only strings under ~6 characters    |" );
-    printf( "\r\n|          can be copied with medium/high reliability    |" );
+    printf( "\r\n| WARNING: Copy-pasting long strings to your serial      |" );
+    printf( "\r\n| -------  terminal may end up in character losses.      |" );
+    printf( "\r\n|          To solve this issue introduce character       |" );
+    printf( "\r\n|          transmit delay in the terminal settings.      |" );
+    printf( "\r\n|          coolTerm:                                     |" );
+    printf( "\r\n|   Options->Transmit->Use transmit character delay      |" );

```

Comment:
:+1: Good change.

Maybe we can reword "may end up in character losses" to "may result in the loss of characters", "introduce character transmit delay" to "use a delay between characters" and "coolTerm:" to "e.g. In coolTerm:".

Just my 2 cents; I'm certainly not the right person correct English grammar :P